code-of-conduct.md

```diff
@@ -0,0 +1,70 @@
+# Code of Conduct
+
+## Summary
+We value the involvement of everyone in the community. We are committed to creating a friendly and respectful place for learning, teaching, and contributing. All participants in our events and communications are expected to show respect and courtesy to others.
+
+To make clear what is expected, everyone participating in Childhood Cancer Data Lab (CCDL) training activities are required to conform to the Code of Conduct. This Code of Conduct applies to all spaces managed by the CCDL including, but not limited to, training workshops, email lists, and online forums such as GitHub, Slack and Twitter.
+
+If you at any time feel harassed or treated inappropriately please contact ccdl@alexslemonade.org.
```

Comment:
```suggestion
If you at any time feel harassed or treated inappropriately, please contact ccdl@alexslemonade.org.
```
And I believe you can either do `<ccdl@alexslemonade.org>` or `[ccdl@alexslemonade.org](mailto:ccdl@alexslemonade.org)` but I would double check that

Reply:
Github-flavored markdown very well may handle the email address as is.